pkgs/development/python-modules/hyperlink/default.nix

```diff
@@ -1,19 +1,26 @@
-{ lib, buildPythonPackage, fetchPypi, isPy27, idna, typing ? null }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, idna
+, typing ? null
+}:
 
 buildPythonPackage rec {
   pname = "hyperlink";
   version = "21.0.0";
```

Comment:
I think this already got updated. Can you please change the commit to ``cleanups, add homepage``?

Reply:
Done, thanks